frontend/device/kindle/device.lua

```diff
@@ -543,6 +543,28 @@ function Kindle:otaModel()
     return model, "ota"
 end
 
+function Kindle:toggleKeyRepeat(toggle)
+    if toggle == true then
+        self.key_repeat[C.REP_DELAY] = 400
+        self.key_repeat[C.REP_PERIOD] = 120
+
+        -- We can't easily clear existing hooks, but we can overwrite the eventAdjustHook
+        -- with the default empty implementation to effectively remove previous hooks
+        self.input.eventAdjustHook = self.input.gestureAdjustHook
```

Comment:
Should be a reset to the *class* `Input.eventAdjustHook` (c.f., `frontend/device/android/device.lua` which does something like this ;)).